aspects_superset_aside/static/html/superset_xblock.html

```diff
@@ -1,8 +1,14 @@
 <div class="superset_xblock">
-    <ul>
-        <li>
-            <a href="superset" target="_blank" rel="noopener noreferrer" class="survey-url"></a>
-        </li>
-    </ul>
-    <button class="button action-primary get-survey-button">Get Survey URL</button>
+    <div>
+        <button class="button action-primary get-survey-button">This is an iframe</button>
+    </div>
+    <iframe
+        width="600"
+        height="400"
+        seamless
+        frameBorder="0"
+        scrolling="no"
+        src="http://superset.local.overhang.io:8088/superset/explore/p/R19Gpb0mQ6D/?standalone=1&height=400"
```

Comment:
@Ian2012 CC @bmtcril  I haven't dug into this possibility, but if we have charts available for everything we want to show in the LMS, then can't simply embed iframes that reference those chart (aka slice) IDs?

E.g. http://superset.local.overhang.io/superset/explore/?form_data={"slice_id":6}&standalone=1 shows the "Events per day" chart in its own window. If you're not logged into Superset then you have to login, but the SSO redirects will get you back there.

cf https://github.com/apache/superset/blob/master/superset/explore/commands/get.py

(For dashboards, there's a few different [standalone](https://superset.apache.org/docs/creating-charts-dashboards/creating-your-first-dashboard/) querystring options, but on/off seems to be the most you get with charts.)

Reply:
I didn't have time to test your idea, but it seems like a feasible solution.

For the instructor dashboard integration, @mariajgrimaldi is working on an openedx filter to allow that integration:

https://github.com/openedx/openedx-filters/pull/96

here is an screenshot of an on-task integration via iframe:

![image](https://github.com/eduNEXT/aspects-superset-aside/assets/33465240/c4345ccf-61a0-462e-abf1-3efdd5490357)

cc @bmtcril

Reply:
We could have a series of instructor-related reports and charts